sample/src/main/java/com/google/accompanist/sample/swiperefresh/SwipeRefreshContentPaddingSample.kt

```diff
@@ -56,6 +58,10 @@ class SwipeRefreshContentPaddingSample : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        // Turn off the decor fitting system windows, which means we need to through handling
+        // insets
+        WindowCompat.setDecorFitsSystemWindows(window, false)
```

Comment:
Isn't this done by the systemUIController library? Or it was previously?

Reply:
No, system ui controller doesn't provide a way to do this.

I think in this particular sample, there was nothing doing this before? So it might have been a bit of a bug with the previous implementation.